src/main/java/frc/team2412/robot/subsystems/IntakeSubsystem.java

```diff
@@ -0,0 +1,64 @@
+package frc.team2412.robot.subsystems;
+
+import com.revrobotics.CANSparkMax;
+import com.revrobotics.CANSparkLowLevel.MotorType;
+
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import static frc.team2412.robot.Hardware.*;
+
+public class IntakeSubsystem extends SubsystemBase{
+	public static final double INTAKE_IN_SPEED = 0.3;
+	public static final double INTAKE_FAST_OUT_SPEED = -0.7;
+
+    CANSparkMax intakeMotorBack;
+    CANSparkMax intakeMotorRight;
+    CANSparkMax intakeMotorFront;
+    CANSparkMax intakeMotorLeft;
+
+    CANSparkMax conveyorBeltOne;
+    CANSparkMax conveyorBeltTwo;
+
+    public IntakeSubsystem() {
+
+        intakeMotorBack = new CANSparkMax(intakeMotorBackID, MotorType.kBrushless);
+        intakeMotorRight = new CANSparkMax(intakeMotorRightID, MotorType.kBrushless);
+        intakeMotorFront = new CANSparkMax(intakeMotorFrontID, MotorType.kBrushless);
+        intakeMotorLeft = new CANSparkMax(intakeMotorLeftID, MotorType.kBrushless);
+
+        conveyorBeltOne = new CANSparkMax(conveyorBeltOneID, MotorType.kBrushless);
+        conveyorBeltTwo = new CANSparkMax(conveyorBeltTwoID, MotorType.kBrushless);
```

Comment:
Need to also configure these motors, preferably by writing and calling method like `resetMotors()` at https://github.com/robototes/ChargedUp2023/blob/f25191718390d11df9ddda789a0f0d5c35686734/src/main/java/frc/team2412/robot/subsystems/IntakeSubsystem.java#L124

Reply:
Completed 👍